modules/eco-setting-node-maintenance-cli.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-setting-node-maintenance-cli_{context}"]
+= Setting a node to maintenance mode in the CLI
+
+Set a node to maintenance mode by marking it as unschedulable and using the `oc adm drain` command to evict or delete pods from the node.
+
+.Procedure
+
+. Mark the node as unschedulable. The node status changes to `NotReady,SchedulingDisabled`.
++
+[source,terminal]
+----
+$ oc adm cordon <node1>
+----
+
+. Drain the node in preparation for maintenance.
++
+[source,terminal]
+----
+$ oc adm drain <node1> --delete-local-data --ignore-daemonsets=true --force
+----
+
+* The `--delete-local-data` flag removes any virtual machine instances on the node that use `emptyDir` volumes. Data in these volumes is ephemeral and is safe to be deleted after termination.
```

Comment:
@razo7 @slintes `--delete-local-data` what does this flag do for the standalone NMO? I was trying to remove references to _virtual machine_

Reply:
That flag is deprecated in the meanwhile, it's named `--delete-emptydir-data` now:
> --delete-emptydir-data=false: Continue even if there are pods using emptyDir (local data that will be deleted when
the node is drained)

I'd suggest:

```
$ oc adm drain <node1> --delete-emptydir-data --ignore-daemonsets=true --force
----

* The `--delete-local-data` flag removes any pods on the node that use `emptyDir` volumes. Data in these volumes is ephemeral and is safe to be deleted after termination.
```